recipes/tiledb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: tiledb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB/archive/{{ version }}.tar.gz
+  sha256: 6e613d222c5f15537dbbd0cf121138c27d480780cc56acd5a7d1ff9f9ce0fe56
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.2
+    - zlib 1.2.11
+    - bzip2
```

Comment:
[Should be pinned](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/3308a5c7667475655899224c111fedf7d69e7db7/recipe/conda_build_config.yaml#L58-L59) to `bzip2 1.0`.